config/locales/client.en.yml

```diff
@@ -1305,6 +1305,7 @@ en:
         invalid: "Please enter a valid email address"
         authenticated: "Your email has been authenticated by %{provider}"
         invite_auth_email_invalid: "Your invitation email does not match the email authenticated by %{provider}"
+        authenticated_by_invite: "Your email has been authenticated by the invite link"
```

Comment:
Copy suggestion: "by the invitation"

Maybe use the existing `user.email.authenticated` two lines above, and add a new translation for just "the invitation" to avoid any inconsistency problems?